apps/laboratory/src/pages/library/solana.tsx

```diff
@@ -30,13 +25,7 @@ const modal = createWeb3Modal({
   termsConditionsUrl: 'https://walletconnect.com/terms',
   privacyPolicyUrl: 'https://walletconnect.com/privacy',
   customWallets: ConstantsUtil.CustomWallets,
-  wallets: [
-    new BackpackWalletAdapter(),
-    new HuobiWalletAdapter(),
-    new PhantomWalletAdapter(),
-    new SolflareWalletAdapter(),
-    new TrustWalletAdapter()
-  ]
+  wallets: [new BackpackWalletAdapter(), new HuobiWalletAdapter(), new SolflareWalletAdapter()]
```

Comment:
Why Phantom and Trust has been removed from the array? @glitch-txs

Reply:
because they support the Wallet Standard, so to properly test the Wallet Standard.